docs/source/adding_new_benchmark.rst

```diff
@@ -0,0 +1,180 @@
+Adding a New Benchmark
+######################
+
+This guide explains how to integrate a new benchmark into **metriq-gym**. You'll learn how to:
```

Comment:
I don't know if I consider this a "learning"

Reply:
True, I reworded it to be:

```
This guide explains how to integrate a new benchmark into **metriq-gym**. The process involves several steps:
```